### DISCUSSION TITLE
Need help following the tutorial here: https://refine.dev/docs/tutorial/getting-started/prepare-env/

### DISCUSSION THREAD

**Discussion by 3MenWalking** [refinedev/refine#5280] [Q&A]
I just followed the steps in the tutorial step by step and picked material UI.  However, when I issue npm run dev, the app fails to start and the error I got is " > tutorial@0.1.0 dev
> refine dev

/Users/wzhang/tutorial/node_modules/node-emoji/lib/index.cjs:47
var import_is2 = __toESM(require("@sindresorhus/is"), 1);
                         ^

Error [ERR_REQUIRE_ESM]: require() of ES Module /Users/wzhang/tutorial/node_modules/node-emoji/node_modules/@sindresorhus/is/dist/index.js from /Users/wzhang/tutorial/node_modules/node-emoji/lib/index.cjs not supported.
Instead change the require of index.js in /Users/wzhang/tutorial/node_modules/node-emoji/lib/index.cjs to a dynamic import() which is available in all CommonJS modules.
    at Object.<anonymous> (/Users/wzhang/tutorial/node_modules/node-emoji/lib/index.cjs:47:26)
    at Object.<anonymous> (/Users/wzhang/tutorial/node_modules/marked-terminal/index.cjs:10:13)
    at Object.<anonymous> (/Users/wzhang/tutorial/node_modules/ink-markdown/build/index.js:9:43)
    at Object.<anonymous> (/Users/wzhang/tutorial/node_modules/@refinedev/cli/dist/cli.js:24:654) {
  code: 'ERR_REQUIRE_ESM'
}".   I have a clean install of nvm/npm/node on a Mac.  Can someone help?

**Comment by BatuhanW** [comment]
Hello @3MenWalking, this is a known issue, and we are working on a fix, it will be released soon. You can follow updates here: https://github.com/refinedev/refine/issues/5279
